src/event_callback.js

```diff
@@ -0,0 +1,46 @@
+import Car from "./car.js";
+import { showElements } from "./elements.js";
+import startRace from "./race.js";
+
+function checkNames(elems) {
+  let result = true;
+  elems.forEach(function (item) {
+    if (item.length > 5) {
+      result = false;
+    }
+  });
+  return result;
+}
+
+export function setNum() {
+  let userInputNum = document.getElementById("#racing-count-input").value;
+  let checkNum = num => !isNaN(num);
+
+  if (checkNum(userInputNum) == false) {
+    alert("잘못된 입력입니다.");
+    document.getElementById("#racing-count-input").value = "";
+  } else {
+    let userInputNames = document.getElementById("#car-names-input").value;
+    const names = userInputNames.split(",");
+    let cars = Array();
```

Comment:
```suggestion
    let cars = new Array();
```
`new Array()` 가 맞는 문법입니다!
`new Array()`로 선언할 때 , 이 붙어서 와서 문제의 소지가 있어서, 보통 new 키워드를 쓰는 방식보다는 리터럴로 넣어주는 방식이 더 선호됩니다. `let cars = []`

https://withhsunny.tistory.com/71

Reply:
앞으론 []이렇게 넣어줘야겠네요